signup-integration-api/src/main/java/io/mosip/signup/api/exception/ProfileException.java

```diff
@@ -0,0 +1,8 @@
+package io.mosip.signup.api.exception;
+
+public class ProfileException extends Exception {
+
+    public ProfileException(String errorCode){
```

Comment:
What are we doing with the errorCode, we should set it to a variable so that errorCode can be used for assertions.